<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TimerMainWindow</class>
 <widget class="QMainWindow" name="TimerMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>475</width>
    <height>378</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>电子时钟</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string/>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLCDNumber" name="lcdHour">
         <property name="digitCount">
          <number>2</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="labelHour">
         <property name="font">
          <font>
           <family>Consolas</family>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="text">
          <string>时</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLCDNumber" name="lcdSecond">
         <property name="digitCount">
          <number>2</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="labelSecond">
         <property name="font">
          <font>
           <family>Consolas</family>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="text">
          <string>分</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLCDNumber" name="lcdMinute">
         <property name="digitCount">
          <number>2</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="labelMinute">
         <property name="font">
          <font>
           <family>Consolas</family>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="text">
          <string>秒</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="title">
       <string/>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0" colspan="2">
        <widget class="QLCDNumber" name="lcdCounter">
         <property name="digitCount">
          <number>12</number>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QPushButton" name="buttonStart">
         <property name="font">
          <font>
           <family>Consolas</family>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="text">
          <string>开始</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QPushButton" name="buttonEnd">
         <property name="font">
          <font>
           <family>Consolas</family>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="text">
          <string>复位</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
